packages/graphback-cli/src/helpers/db.ts

```diff
@@ -52,10 +53,17 @@ export const createDBResources = async(): Promise<void> => {
       await execa('touch', ['db.sqlite'])
     }
 
-    const path: string = process.cwd()
-    const schemaText: string = models.found.map((m: string) => readFileSync(`${path}/${m}`, 'utf8')).join('\n')
+     const modelDefinitions: IGraphbackModel[] = models.found.map((m: string) => {
+      const name = path.posix.basename(`${process.cwd()}/${m}`, '.graphql');
```

Comment:
Is there any reason why you specifically needed to use `path.posix`? From the [Node Docs](https://nodejs.org/api/path.html#path_path_posix) I see that `The path.posix property provides access to POSIX specific implementations of the path methods.`

I may be missing something but Ideally we shouldn't need it and it could introduce problems later on.

Reply:
There was a reason I chose it, but that reason was wrong. I thought POSIX was for making `path` work on both Windows and Linux/Unix systems. Looking at the docs again I see that I just misunderstood the documentation. Thanks :+1: